app/components/container/container.html

```diff
@@ -35,7 +35,14 @@
           <tbody>
             <tr>
               <td>ID</td>
-              <td>{{ container.Id }}</td>
+              <td ng-if="!container.toggleId">
+                {{ container.Id.slice(0,12) }}
```

Comment:
It would better to use [truncate](https://github.com/portainer/portainer/blob/develop/app/components/images/images.html#L127) filter because it adds dots at the end and overall more robust.